unit-tests/live/safety/test-safety-metadata.py

```diff
@@ -0,0 +1,42 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#test:device D585S
```

Comment:
Please add: #test:donotrun
Until we will connect the SC to LibCI

Reply:
ok